torch/ao/quantization/observer.py

```diff
@@ -397,6 +397,7 @@ def __init__(
         dtype=torch.quint8,
         qscheme=torch.per_tensor_affine,
         reduce_range=False,
+        memoryless=False,
```

Comment:
nit: a new keyword argument is safer to add to the end of the argument list, to avoid breaking user code